site/layouts/shortcodes/partial.html

```diff
@@ -1 +1 @@
-{{ partial (.Get 0) . }}
+{{- partial (.Get 0) . -}}
```

Comment:
Why this change? If it's really needed (and I have to confirm myself later), it should be made in a separate PR.

Reply:
In this PR, I use the partial to include code in the codesnippets. Without these dashes, a newline is added to the example code snippet. Only adding the last dash can be enough I think, since there is no leading newline.

Reply:
Please move this change to a separate PR and I'll have a look :)

Reply:
See https://github.com/twbs/bootstrap/pull/30472#issue-394832461

Reply:
Made a custom shortcode, which seems to fix the issue and decrease the code in `tables.md` even more.